packages/tre/src/runtime/index.ts

```diff
@@ -40,13 +52,14 @@ class NativeRuntime implements Runtime {
   readonly #args: string[];
 
   #process?: childProcess.ChildProcess;
-  #processExitPromise?: Promise<number>;
+  #processExitPromise?: Promise<void>;
 
   constructor(
-    protected readonly runtimeBinaryPath: string,
-    protected readonly entryPort: number,
-    protected readonly loopbackPort: number
+    runtimeBinaryPath: string,
+    entryPort: number,
+    loopbackPort: number
   ) {
+    super(runtimeBinaryPath, entryPort, loopbackPort);
```

Comment:
Is this super call needed? `Runtime` doesn't do anything with its arguments, right?